python/python/glide/async_commands/core.py

```diff
@@ -1405,6 +1422,57 @@ async def type(self, key: str) -> str:
         """
         return cast(str, await self._execute_command(RequestType.Type, [key]))
 
+    async def geoadd(
+        self,
+        key: str,
+        members_coordinates: Mapping[str, Coordinate],
+        existing_options: Optional[ConditionalChange] = None,
+        changed: bool = False,
```

Comment:
naming is a bit confusing, but I can't propose anything better

Reply:
please change 'changed' to 'return_changed_count' or 'return_changed'